lib/accessory.js

```diff
@@ -9,34 +9,44 @@ let Service, Characteristic;
 
 class Accessory {
 
-  constructor (raw, overrides) {
+  constructor (raw, overrides, log) {
     assertString(raw.powerstrip_id, '`raw.powerstrip_id` must be set');
     assertString(raw.name, '`raw.name` must be set');
     assertString(raw.device_manufacturer, '`raw.device_manufacturer` must be set');
     assertString(raw.model_name, '`raw.model_name` must be set');
     assertString(raw.serial, '`raw.serial` must be set');
     assertArray(raw.outlets, '`raw.outlets` must be an array');
     assertArray(overrides, "`overrides` must be an array");
-
+	
+	this.log = log;
     this.id = raw.powerstrip_id;
     this.name = raw.name;
     this.manufacturer = raw.device_manufacturer;
     this.model = raw.model_name;
     this.serialNumber = raw.serial;
+	this.log("Strip " + this.name + " ('" + this.model + " - S/N '" + this.serialNumber + "') initialized.");
+	
+	var that = this;
+	
     this.outlets = raw.outlets.map(outlet => {
       const override = this.outlet(overrides, outlet.uuid) || {};
       const outletConfig = Object.assign({
         uuid: outlet.uuid,
         name: outlet.name,
-        hapService: 'Outlet'
+        hapService: 'Outlet',
+		log : this.log
```

Comment:
Yeah this was probably cleanup I missed while I was playing with passing the homebridge log object somewhere that this plugin could use for the outlets.  I can remove.